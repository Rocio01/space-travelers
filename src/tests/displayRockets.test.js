import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import DisplayRockets from '../components/dislayRockets';
import store from '../redux/configureStore';

test('matches DisplayRockets component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <DisplayRockets />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
