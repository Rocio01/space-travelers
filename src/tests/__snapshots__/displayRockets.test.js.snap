// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches DisplayRockets component snapshot 1`] = `
<div
  className="container"
/>
`;

exports[`matches Rockets component snapshot 1`] = `
<div
  className="container"
/>
`;
